diesel/src/pg/query_builder.rs

```diff
@@ -27,8 +22,10 @@ impl QueryBuilder<Pg> for PgQueryBuilder {
     }
 
     fn push_identifier(&mut self, identifier: &str) -> BuildQueryResult {
-        let escaped_identifier = try!(self.conn.escape_identifier(identifier));
-        Ok(self.push_sql(&escaped_identifier))
+        self.push_sql("\"");
+        self.push_sql(&identifier.replace('"', "\"\""));
+        self.push_sql("\"");
```

Comment:
![](https://i.imgflip.com/1hc54t.jpg)

(This broke the first meme generator I tested)